config/environments/review.rb

```diff
@@ -5,4 +5,11 @@
 Rails.application.configure do
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
+
+  # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
+  config.force_ssl = true
+  config.ssl_options = {
+    hsts: { preload: true, subdomains: true, expires: 1.year },
```

Comment:
I'm not sure whether something hasn't rolled out yet, but if I look at the header in the review app the expiry seems to be 0.5 year rather than 1 year 🤔 
Will the `subdomains` directive work for review apps because they are already subdomains of `teacherservices.cloud`?

Reply:
That makes sense for some things to be managed at a higher level.

Reply:
Thanks for checking @stevehook - I'll have to check this with FIDUS and ask them about what remediation they recommend. I can see in the thread that DevOps say they keep on reporting this. Could you assign the Trello ticket to me please. Thanks.